packages/orchestrator-ui-components/src/components/WfoSubscription/WfoSubscriptionActions/WfoSubscriptionActions.tsx

```diff
@@ -134,23 +149,24 @@ export const WfoSubscriptionActions: FC<WfoSubscriptionActionsProps> = ({
             const tooltipContent = t(action.reason, flattenArrayProps(action));
 
             return (
-                <div>
+                <div css={tooltipMenuItemStyle}>
                     <EuiToolTip position="top" content={tooltipContent}>
-                        {actionItem}
+                        <WfoSubscriptionActionExpandableMenuItem
+                            subscriptionAction={action}
+                            onClickLockedRelation={() => setPopover(false)}
```

Comment:
Doesn't the component expect a function with a (relation: string) signature? I think there's a type mismach here

Reply:
removed the unused parameter